lib/chef/resource/powershell_script.rb

```diff
@@ -46,6 +44,8 @@ def convert_boolean_return(arg=nil)
       def self.get_default_attributes(opts)
         {:convert_boolean_return => true}
       end
+
+      private_class_method :get_default_attributes
```

Comment:
Yeah, I got that part, but it seems like someone intended it to be protected rather than private so I'm curious if there was a reason for that.